common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -95,20 +95,13 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
   }
 
   _renderPagination(): ?React$Node {
-    if (this.state.current_page === this.state.project_pages) {
+    if ((this.state.current_page === this.state.project_pages) || this.state.projects_loading) {
```

Comment:
I think it's still useful to show Loading... after the user clicks on the button.

Reply:
That makes sense. It's hard for me to test that locally because my data loads basically instantly but I wrote a conditional that should handle it. My thought was if this.state.projects isn't empty (so it's not showing the button on initial pageload) and the loading state is true, it should show the "Loading..." button in a disabled state.

When I manually set the state this way in React it worked, but it needs testing with live data.